pkg/controller/cassandrarestore/status.go

```diff
@@ -0,0 +1,54 @@
+package cassandrarestore
+
+import (
+	"context"
+	"fmt"
+
+	api "github.com/Orange-OpenSource/casskop/pkg/apis/db/v1alpha1"
+	"github.com/go-logr/logr"
+	apierrors "k8s.io/apimachinery/pkg/api/errors"
+	"k8s.io/apimachinery/pkg/types"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+func UpdateRestoreStatus(c client.Client, restore *api.CassandraRestore, status api.CassandraRestoreStatus, reqLogger logr.Logger) error {
+	typeMeta := restore.TypeMeta
+
+	restore.Status = status
+
+	err := updateRestoreStatus(c, restore)
+
+	if err != nil {
```

Comment:
return here if there is an error instead

Reply:
But with this way we are able to retry in case of Conflict update instead of requeueing ?